Networking/Networking/Remote/OrdersRemote.swift

```diff
@@ -125,6 +125,30 @@ public class OrdersRemote: Remote {
         enqueue(request, mapper: mapper, completion: completion)
     }
 
+    /// Updates the specified fields of a given order.
+    ///
+    /// - Parameters:
+    ///     - siteID: Site which hosts the Order.
+    ///     - order: Order to be updated.
+    ///     - fields: Fields from the order to be updated.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func updateOrder(from siteID: Int64, order: Order, fields: [UpdateOrderField], completion: @escaping (Result<Order, Error>) -> Void) {
+        let path = "\(Constants.ordersPath)/\(order.orderID)"
+        let mapper = OrderMapper(siteID: siteID)
+        let parameters: [String: AnyHashable] = {
+            fields.reduce(into: [:]) { params, field in
+                switch field {
+                case.customerNote:
```

Comment:
nit: missing a space

```suggestion
                case .customerNote:
```